store/src/java/com/zimbra/cs/ephemeral/migrate/AttributeMigrationUtil.java

```diff
@@ -187,7 +187,8 @@ private static void showMigrationInfo() throws ServiceException {
     private static void usage() {
         HelpFormatter format = new HelpFormatter();
         format.printHelp(new PrintWriter(System.err, true), 80,
-            "zmmigrateattrs [options] [URL] [attr1 attr2 attr3 ...]", null, OPTIONS, 2, 2, null);
+            "zmmigrateattrs [options] [URL] [attr1 attr2 attr3 ...]", null, OPTIONS, 2, 2,
+            "\n'URL' MUST be provided for all options except for '--clear','-c', '--status' and '-s'");
```

Comment:
nit: no space between comma (,) and before '-c'
some possible minor wording tweaks:
    ... except for '--clear' (-c) and '--status' (-s)
or ... except for '--clear|-c' and '--status|-s'

Reply:
I'll go with the first suggestion.  Didn't much like the wording I had come up with but couldn't think of a good alternative - so this is nice!